GLMakie/src/precompiles.jl

```diff
@@ -49,7 +49,6 @@ let
             screen = Screen(Scene(), config; visible=false, start_renderloop=false)
             close(screen)
 
-            empty!(atlas_texture_cache)
```

Comment:
Why was this removed? I guess it's correct, since it doesn't seem to error, but just want to make sure.

Reply:
This would empty the dict without doing OpenGL or Makie cleanup

Reply:
I think that was the idea